addons/l10n_account_edi_ubl_cii_tests/tests/common.py

```diff
@@ -225,7 +225,7 @@ def _generate_move(self, seller, buyer, send=True, **invoice_kwargs):
             account_move._generate_pdf_and_send_invoice(self.move_template)
         return account_move
 
-    def _assert_invoice_attachment(self, attachment, xpaths, expected_file):
+    def _assert_invoice_attachment(self, attachment, xpaths, expected_file_path):
```

Comment:
There are 4 commits in the PR. This change is in its own commit: https://github.com/odoo/odoo/pull/128049/commits/49b7258a2ef083fc73913f4453864d22190ddd1a

Reply:
I noticed the different commits but I think instead of just saying `file and file_path: clarify which is what`, say that you renamed variables to clarify. It's not much but it explains more why you changed that (and that it's not related to first aim of the commit)